akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -356,10 +356,10 @@ of code should only run for a specific role.
 @@snip [StatsSampleSpec.scala](/akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsSampleSpec.scala) { #test-statsService }
 
 Once again we take advantage of the facilities in @ref:[testkit](testing.md) to verify expected behavior.
-Here using `testActor` as sender (via `ImplicitSender`) and verifying the reply with `expectMsgPF`.
+Here using `testActor` as sender (via @scaladoc[ImplicitSender](akka.testkit.ImplicitSender)) and verifying the reply with `expectMsgType`.
```

Comment:
Looks like a little typo. Should be `expectMsgType`. Am I wrong?
Should I add link for it?

Reply:
I suspect we changed the example but forgot to also change the text (https://doc.akka.io/docs/akka/current/cluster-usage.html). Would be nice to add a link but this seems OK to me as well.